Src/BootCamp.Chapter/Items/Armpiece.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter.Items
+{
+    public class Armpiece
+    {
+        private string _name;
+        private decimal _price;
+        private float _weight;
```

Comment:
Codacy found an issue: [Make '_weight' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)